crm_compassion/models/crm_lead.py

```diff
@@ -11,13 +11,14 @@
 from odoo.addons.crm.models.crm_lead import CRM_LEAD_FIELDS_TO_MERGE
 
 from odoo import api, models, fields
-
+import datetime
 
 class CrmLead(models.Model):
     _inherit = "crm.lead"
 
     planned_sponsorships = fields.Integer(
         "Expected new sponsorships", track_visibility="onchange"
+        ,compute="_compute_planned_sponsorship"
```

Comment:
If we store it, we can search with this, order and group by, which could be helpful. It will also display the kanban view faster because it won't need to compute all values when opening the view.
```suggestion
        ,compute="_compute_planned_sponsorship", store=True
```